src/actions/sources/newSources.js

```diff
@@ -41,14 +41,23 @@ function createOriginalSource(
   };
 }
 
+// TODO: It would be nice to make getOriginalURLs a safer api
```

Comment:
FWIW the toolbox wraps this in a proxy that reports errors to the console.

Reply:
great